src/components/PDF/ApplicationForm.tsx

```diff
@@ -51,7 +52,8 @@ class ApplicationForm extends Component<Props, State> {
       formQuestions: undefined,
       formAnswers: {},
       pdfApplication: undefined,
-      buttonState: '',
```

Comment:
Why was buttonState removed?

Reply:
Well, it is unused I believe because now we are transitioning to pagination on the questions, and so the button does not swirl (I got rid of the swirling button). I think we need to redo it now with pagination (Vanessa covered this)